<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupervisionClass</class>
 <widget class="QMainWindow" name="SupervisionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Supervision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_TitleSupervision">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>22</pointsize>
      <weight>9</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 22pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>SUPERVISION DE LA BARRIERE</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1121</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="interface_accueil">
     <widget class="QWidget" name="widget_Accueil" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>60</y>
        <width>1121</width>
        <height>431</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_FormConnect" native="true">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>-10</y>
         <width>501</width>
         <height>441</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_TitleConnexion">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>251</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>16</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 16pt &quot;MS Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Connexion à votre espace</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Login">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: italic 13pt &quot;MS Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_Login">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Mdp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: italic 13pt &quot;MS Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>Mot de passe</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_Mdp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>210</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_AccesConnexion">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>290</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 15px;
background-color: rgb(0, 60, 202);
border: 2px solid white;
padding: 5px;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Accéder</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_ErrorConnect">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>250</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="img_key">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>40</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/Supervision/ressources/key.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_Login</zorder>
       <zorder>edit_Login</zorder>
       <zorder>label_Mdp</zorder>
       <zorder>edit_Mdp</zorder>
       <zorder>btn_AccesConnexion</zorder>
       <zorder>label_ErrorConnect</zorder>
       <zorder>label_TitleConnexion</zorder>
       <zorder>img_key</zorder>
      </widget>
      <widget class="QLabel" name="img_LogoMonitoring">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>371</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Supervision/ressources/monitoring.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="interface_supervision">
     <widget class="QWidget" name="widget_Supervision" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1121</width>
        <height>501</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_TitleInfos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Informations des Équipements</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_AskStatut">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>430</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 15px;
border: 2px solid white;
background-color: rgb(3, 144, 3);
color: white;</string>
       </property>
       <property name="text">
        <string>Renvoyer une demande d'état</string>
       </property>
      </widget>
      <widget class="QLabel" name="img_camera">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>91</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Supervision/ressources/img_camera.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_camera">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Statut Caméra : </string>
       </property>
      </widget>
      <widget class="QLabel" name="img_arduino1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Supervision/ressources/img_arduino.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="img_arduino2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>330</y>
         <width>101</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Supervision/ressources/img_arduino.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_ard1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>250</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Statut Arduino 1 : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ard2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>360</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Statut Arduino 2 : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_statutcamera">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_statutard1">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>250</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_statutard2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>360</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_BienvenueTexte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Bienvenue sur l'interface de supervision de la barrière et de ses équipements.</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_LaProLogo">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Supervision/ressources/logo_lapro.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_Accueil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="background_white" native="true">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-230</y>
      <width>1231</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <zorder>background_white</zorder>
   <zorder>label_TitleSupervision</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label_BienvenueTexte</zorder>
   <zorder>img_LaProLogo</zorder>
   <zorder>line_Accueil</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Supervision.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_AccesConnexion</sender>
   <signal>clicked()</signal>
   <receiver>SupervisionClass</receiver>
   <slot>on_btnAccesConnexion_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnAccesConnexion_clicked()</slot>
 </slots>
</ui>
